doc/source/rllib/rllib-learner.rst

```diff
@@ -0,0 +1,459 @@
+:py:class:`~ray.rllib.core.learner.learner_group.LearnerGroup` API. Under this paradigm, 
+the :py:class:`~ray.rllib.core.learner.learner_group.LearnerGroup` maintains multiple 
+copies of the same :py:class:`~ray.rllib.core.learner.learner.Learner` with identical 
+parameters and hyperparameters. Each of these
+:py:class:`~ray.rllib.core.learner.learner.Learner` instances computes the loss and gradients on a
+shard of a sample batch and then accumulates the gradients across the 
+:py:class:`~ray.rllib.core.learner.learner.Learner` instances. Learn more about data-distributed 
+parallel learning in 
+`this article. <https://pytorch.org/tutorials/intermediate/ddp_tutorial.html>`_
+
+:py:class:`~ray.rllib.core.learner.learner_group.LearnerGroup` also allows for 
+asynchronous training and (distributed) checkpointing for durability during training.
+
+Enabling Learner API in RLlib experiments
+=========================================
+
+Adjust the amount of resources for training using the 
+`num_gpus_per_learner_worker`, `num_cpus_per_learner_worker`, and `num_learner_workers`
+arguments in the `AlgorithmConfig`.
+
+.. testcode::
+	:hide:
+    :skipif: True
+
+    from ray.rllib.algorithms.ppo.ppo import PPOConfig
+
+.. testcode::
+    :skipif: True
+
+    config = (
+        PPOConfig()
+        .resources(
+            num_gpus_per_learner_worker=0,  # Set this to 1 to enable GPU training.
+            num_cpus_per_learner_worker=1,
+            num_learner_workers=0  # Set this to greater than 0 to allow for DDP style 
+                               # updates.
+        )
+        .training(_enable_learner_api=True)
+        .rl_module(_enable_rl_module_api=True)
+    )
+
+.. testcode::
+	:hide:
+    :skipif: True
+
+	from gymnasium.envs.classic_control.cartpole import CartPoleEnv
+
+	config = config.environment(env=CartPoleEnv)
+	config.build()  # test that the algorithm can be built with the given resources
+
+
+.. note::
+    
+    This features is in alpha. If you migrate to this algorithm, enable the feature by 
+    setting `_enable_learner_api` and `_enable_rl_module_api` flags in the 
+    `AlgorithmConfig`.
+
+    The following algorithms support `Learner` out of the box. Implement
+    an algorithm with a custom `Learner` to leverage this API for other algorithms.
+
+    .. list-table::
+       :header-rows: 1
+       :widths: 60 60
+
+       * - Algorithm
+         - Supported Framework
+       * - **PPO**
+         - |pytorch| |tensorflow|
+       * - **Impala**
+         - |pytorch| |tensorflow|
+       * - **APPO**
+         - |pytorch| |tensorflow|
+
+
+Basic Usage
+===========
+
+Use the `LearnerGroup` utility to interact with multiple learners. 
+
+Construction
+------------
+
+If you enable the `RLModule`
+and `Learner` APIs via the RLlib algorithm config, then `Algorithm` constructs a `LearnerGroup` for you, but if you’re using these APIs standalone, you can construct the `LearnerGroup` as follows.
```

Comment:
```suggestion
and `Learner` APIs via the `AlgorithmConfig`, then calling `AlgorithmConfig.build()`, construct a `LearnerGroup` for you, but if you’re using these APIs standalone, you can construct the `LearnerGroup` as follows.
```

Algorithm is such an overloaded term. I'd not expect all users to know that PPO is an instance of Algorithm and see through this software engineering perspective of things. Also, we should link to API refs here again imho

Reply:
all done